README.md

```diff
@@ -1 +1,80 @@
-# Hello_Microverse_Project
\ No newline at end of file
+# Hello_Microverse_Project
+
+This project is learning setup for microverse student on how
+
+- Create branches with different base branches.
+- Follow Gitflow.
+- Understand the concept of version control.
+- Open pull requests in GitHub.
+- Write descriptive, easy to understand commit messages.
+- Write short, detailed, easy to understand descriptions of pull requests.
+- Write a descriptive and easy to understand README in business English for every project submitted.
+- Demonstrate an ability to submit a project for code review.
+
+![screenshot of the site](/img/screen_shot.PNG)
+
+In this project, you will set up a "Hello world" repository
+
+- Added three files that include html file, css file and test.md file
+- Set up linters files for Html and css
+  And I have made sure that the last file is ignored by gitignore
+
+# BUIT WITH
+
+- HTML
+- CSS
+- GIT
+
+# LIVE DEMO
```

Comment:
- [ ] Since you have no live demo yet, it would probably be better to remove this sub-heading, as it is somewhat unnecessary.